OWNERS

```diff
@@ -2,9 +2,10 @@
 
 reviewers:
 - MushuEE
+- spiffxp
```

Comment:
FWIW I'm willing to go back to approver if you're OK with it

Reply:
I opened https://github.com/kubernetes-sigs/kubetest2/pull/215 to add me as approver

We'll need to get @richackard added as an org member